src/components/WhoWeAre/Bios.tsx

```diff
@@ -0,0 +1,241 @@
+const Bios = [
```

Comment:
Since this is not a component, it's just data, I would rename the file: `bios.ts` and rename this const to lowercase `bios` as well